modules/core/src/v2/keychains.ts

```diff
@@ -120,46 +128,28 @@ Keychains.prototype.updatePassword = function(params, callback) {
  * @param callback
  * @returns {object}
  */
-Keychains.prototype.updateSingleKeychainPassword = function({ keychain, oldPassword, newPassword }) {
-  if (!_.isString(oldPassword)) {
-    throw new Error('expected old password to be a string');
-  }
-
-  if (!_.isString(newPassword)) {
-    throw new Error('expected new password to be a string');
-  }
+Keychains.prototype.updateSingleKeychainPassword = function( params: { keychain: any, oldPassword: string, newPassword: string }): any {
```

Comment:
i dont think this a breaking change is it?

Reply:
No it's not, but please extract into interface. Can we do any better than `any` for the return type?